pre_commit_hooks/util.py

```diff
@@ -7,6 +7,9 @@
 from typing import Set
 
 
+KNOWN_BINARY_FILE_EXTS = ('.pdf',)
```

Comment:
this shouldn't be necessary, `identify` will filter out binary files if you use `types: [text]`